gridded_etl_tools/dataset_manager.py

```diff
@@ -283,10 +283,15 @@ def extract(self, date_range: tuple[datetime.datetime, datetime.datetime] | None
             )
         self.new_files = []
 
-    def transform(self):
+    def transform_data_on_disk(self):
```

Comment:
After looking at managers I'm wondering if this function could be named "virtualize"

Reply:
It seems OK to me if the Zarr JSON is referred to as the virtual Zarr even when it's not loaded into memory, and isn't that how `zarr_json_to_dataset` describes the virtual Zarr?
```
def zarr_json_to_dataset(self, zarr_json_path: str = None, decode_times: bool = True) -> xr.Dataset:
     """
     Open the virtual zarr at `self.zarr_json_path()` and return as a xr.Dataset object after applying
     any desired postprocessing steps
```
But I should have mentioned the reason I bring it up is that having both `transform_data_and_disk` and `transform_dataset_in_memory` start with the word transform is confusing at least to me. One rewrites files on disk and creates JSON, and the other returns an xr.Dataset. I think when we refactor transform to be its own class we'll further improve this situation, but clearer distinction between the two new transform function names might be something good to have for now.